gluefactory/train.py

```diff
@@ -48,11 +48,12 @@
     "optimizer_options": {},  # optional arguments passed to the optimizer
     "lr": 0.001,  # learning rate
     "lr_schedule": {
-        "type": None,
+        "type": None,  # can be any of [factor, exp] or torch.optim.lr_scheduler
         "start": 0,
         "exp_div_10": 0,
         "on_epoch": False,
         "factor": 1.0,
+        "options": None,  # add lr_scheduler arguments here
```

Comment:
Shouldn't this be `{}` by default?

Reply:
Yes that would be more clean.

```suggestion
        "options": {},  # add lr_scheduler arguments here
```